src/test/java/com/wikia/webdriver/testcases/discussions/OpenGraphTests.java

```diff
@@ -52,6 +52,14 @@ public void validLinkGeneratesOpenGraphItemInNewPostOnDesktop() throws Malformed
     verifyOpenGraphInNewPost(page, postsCreator);
   }
 
+  @Test
+  @InBrowser(emulator = Emulator.DESKTOP_BREAKPOINT_BIG)
+  public void validOpenGraphItemCanBeSubmittedInNewPostOnDesktop() throws MalformedURLException {
```

Comment:
imo we should indicate that there is no description in test name

sth like `validOpenGraphItemCanBeSubmittedWithoutDescriptionInNewPostOnDesktop()`
as well as `validOpenGraphItemCanBeSubmittedWithDescriptionInNewPostOnDesktop()` to the previous one

the same applied to the rest of test cases